core/src/main/java/org/testcontainers/DockerClientFactory.java

```diff
@@ -96,8 +107,71 @@ public DockerClient client() {
                     "  Operating System: " + dockerInfo.getOperatingSystem() + "\n" +
                     "  Total Memory: " + dockerInfo.getMemTotal() / (1024 * 1024) + " MB");
 
-            checkVersion(version.getVersion());
-            checkDiskSpaceAndHandleExceptions(client);
+            if (!TestcontainersConfiguration.getInstance().getDisableChecks()) {
+                VisibleAssertions.info("Checking the system...");
+
+                VisibleAssertions.assertThat("Docker version", version.getVersion(), new BaseMatcher<String>() {
+                    @Override
+                    public boolean matches(Object o) {
+                        return new ComparableVersion(o.toString()).compareTo(new ComparableVersion("1.6.0")) >= 0;
+                    }
+
+                    @Override
+                    public void describeTo(Description description) {
+                        description.appendText("is newer than 1.6.0");
+                    }
+                });
```

Comment:
Couldn't this be a bit shorter by just using an `assertTrue` method? An anonymous inner class to satisfy `assertThat` feels a bit heavy..?